doc/src/manual/constructors.md

```diff
@@ -381,28 +376,27 @@ Point{Float64}
 
 However, other similar calls still don't work:
 
-```julia
+```jldoctest parametric2
 julia> Point(1.5,2)
-ERROR: MethodError: no method matching Point{T<:Real}(::Float64, ::Int64)
+ERROR: MethodError: no method matching Point(::Float64, ::Int64)
 Closest candidates are:
-  Point{T<:Real}{T<:Real}(::T<:Real, !Matched::T<:Real) at none:2
-  Point{T<:Real}{T}(::Any) at sysimg.jl:66
- ...
+  Point{T}(::Any) at sysimg.jl:24
+  Point{T<:Real}(::T<:Real, ::T<:Real) at none:1
```

Comment:
There is a fix in the making in Documenter.jl. Let's just ignore color stuff until it is merged imo.